src/components/ResourceCard/dropdown.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter, Link } from 'react-router-dom';
+import Swal from 'sweetalert2';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Dropdown } from 'react-bootstrap';
+import { deleteResourceAction } from 'store/actions/resource';
+import { cloneActivity } from 'store/actions/search';
+import ResourceCardDropdownShare from './shareResource';
+
+import './style.scss';
+
+const ResourceCardDropdown = (props) => {
+  const {
+    resource,
+    deleteResource,
+    playlist,
+    match,
+  } = props;
+
+  const handleDelete = (e) => {
+    e.preventDefault();
+    Swal.fire({
+      title: 'Are you sure you want to delete this activity?',
+      showDenyButton: true,
+      showCancelButton: true,
+      confirmButtonText: 'Yes',
+      denyButtonText: 'No',
+    }).then((result) => {
+      if (result.isConfirmed) {
+        deleteResource(resource.id);
+      }
+    });
+  };
+
+  return (
+
```

Comment:
Remove blank line

Reply:
done

Reply:
I mean the blank line after open bracket, not before return